data/brands/amenity/fast_food.json

```diff
@@ -495,7 +495,20 @@
       }
     },
     {
-      "displayName": "Barburrito",
+      "displayName": "Barburrito (CA)",
+      "id": "barburrito-e49d2e",
+      "locationSet": {"include": ["ca"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Barburrito",
```

Comment:
The seem to case it as BarBurrito on
[Facebook](https://www.facebook.com/BarBurritoCanada)
and the footer on the [website](https://www.barburrito.ca/location/toronto-lawrence-square/).

So I vote for that, unless you have a strong opinion otherwise.

Reply:
I changed Barburrito to BarBurrito